app/locales/ja/translations.ts

```diff
@@ -259,7 +259,7 @@ export default {
         status_link_text: 'ステータスページ',
         status_paragraph_2: '私たちのサービスがどのように動作しているかについての更新情報。',
         consultation_title: '統計コンサルティングをお探しですか',
-        consultation_paragraph: 'COSは統計コンサルティングを無料で提供しています。 このサービスの詳細については、',
+        consultation_paragraph: 'COSは統計コンサルティングを提供します。 このサービスの詳細については、',
```

Comment:
@Yusuke-KOMIYAMA can you confirm that this is correct?

Reply:
I am sorry that it will be delayed to notice sorry. The changed part of the Japanese translation file is correct.

Reply:
No worries. Thank you for confirming it.